bundles/org.openhab.binding.pulseaudio/src/main/java/org/openhab/binding/pulseaudio/internal/PulseAudioBindingConfiguration.java

```diff
@@ -0,0 +1,52 @@
+/**
+ * Copyright (c) 2010-2021 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.pulseaudio.internal;
+
+import java.util.HashSet;
+import java.util.Set;
+
+/**
+ * Contains the binding configuration
+ *
+ * @author Gwendal Roulleau - Initial contribution
+ *
+ */
+public class PulseAudioBindingConfiguration {
```

Comment:
Please add `@NonNullByDefault`

Reply:
Damn, again ! I hope I will remember it next time. Sorry.